docs/src/charm/reference/proxy.md

```diff
@@ -1,2 +1,4 @@
-```{include} /snap/reference/proxy.md
+# Proxies
+
```

Comment:
Same here again. Do you want to rename it?
![image](https://github.com/user-attachments/assets/6fef3f41-6f79-46bd-bd59-01a8dcff1c90)